packs/spells/vector-screen.json

```diff
@@ -25,12 +25,14 @@
                 "5": {
                     "area": {
                         "areaType": "square",
+                        "type": "burst",
                         "value": "10"
                     }
                 },
                 "6": {
                     "area": {
                         "areaType": "square",
+                        "type": "burst",
                         "value": "10"
```

Comment:
This looks like a bug in the heightening overlay. @CarlosFdez any thoughts?

Reply:
I think it's legacy data that slipped in. It's be good to remove it.

Reply:
Anything I should do with this? It came out like that by itself so I assumed it was something the system did when I unlocked the compendium to edit the spell. It looked correct according to the spell's description, so I left it alone.

Reply:
Value and type are the correct fields, so areaType is legacy data. I don't see any reference to it in the templates.